topdown/bindings.go

```diff
@@ -104,6 +107,9 @@ func (u *bindings) plugNamespaced(a *ast.Term, caller *bindings) *ast.Term {
 		})
 		return &cpy
 	case ast.Set:
```

Comment:
For the other three shortcuts added here, I have not yet thought about when they'd happen -- they do seem to make sense in a pattern-match kind of way 🙃